citi-onboarding/HighBall
# Catalog

## What I did

- Created catalog section
- Catalog component
- Card component
- Implemented an slider
   
## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout catalog`
   1. `git pull origin catalog`
   1. `yarn install`
   1. `code .`
   1. `cd client`
   1. `yarn start`

1. In your browser go to http://localhost:3000

1. In VScode open `./client`

1. Read the changed files section on GitHub 

1. Compare the page at 1366px width with the [mockup](https://www.figma.com/file/FTlYITKEgwlp7rDJzX6IBc/Squad-Julia?node-id=189%3A6)

1. Check if the background bubbles keep it's size and increase in amount as the screen expands

1. Test the things that you think are worth testing, even the ones that are not in this description
